### DISCUSSION TITLE
Add resource template for One-To-One relationship

### DISCUSSION THREAD

**Discussion by KiaBoluki** [filamentphp/filament#6945] [Ideas]
Hello @danharrin,

I hope this message finds you well. I have a feature suggestion for Filament that I believe would greatly enhance its functionality. It would be wonderful if Filament could support One-To-One relationships and offer a resource specifically designed for managing them. My suggestion includes the addition of the following pages:

- Create (if not already exists)
- Edit
- View (to replace the index page list)

I truly believe that this feature is essential as I have personally encountered challenges while attempting to create a Filament page template for `user.profile`. After several days of searching for a solution, I came across this discussion: 

https://github.com/filamentphp/filament/discussions/6116

Creating a custom page seems to be the workaround, but dealing with links, records, and relations can be quite complex and overwhelming, at least in my experience.

Thank you very much for considering this request. I would be grateful for any guidance or assistance you can provide in implementing this feature.

Please feel free to let me know if you require any further information or clarification.

Best regards,

**Comment by danharrin** [comment]
What is your use case for this? I feel like it can be simply accomplished with a custom page and the form builder
